lib/tasks/export.rake

```diff
@@ -0,0 +1,26 @@
+require 'csv'
+
+namespace :export do
+  def fetch_list(list_id)
+    begin
+      list = SubscriberList.find(list_id)
+      { subscriber_list_id: list_id, title: list.title, count: list.subscribers.count }
+    rescue ActiveRecord::RecordNotFound
+      warn "could not fetch record for #{list_id}"
+    end
+  end
+
+  desc "Export the number of subscribers for a collection of lists as a csv, accepts multiple arguments"
+  task :csv, [:subscriber_list_id] => :environment do |_, args|
+    CSV($stdout, headers: %i[subscriber_list_id title count], write_headers: true) do |csv|
+      rows = args.to_a.map { |list_id| fetch_list(list_id) }
+      rows.each { |row| csv << row unless row.nil? }
```

Comment:
`rows.compact` would give you all the rows that aren't `nil` so you might be able to make it more succinct with that 😃